frontend/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM docker.io/library/node:20-bookworm-slim AS build
+
+WORKDIR /app
+
+COPY ./yarn.lock /app/yarn.lock
+COPY ./package.json /app/package.json
+COPY ./dashboard/package.json /app/dashboard/package.json
+COPY ./relay-workflows-lib /app/relay-workflows-lib
+COPY ./workflows-lib /app/workflows-lib
+RUN yarn install --frozen-lockfile
+
+COPY . .
+WORKDIR /app/relay-workflows-lib
+RUN yarn run relay
+
+WORKDIR /app/dashboard
+RUN yarn run relay && yarn tsc && yarn build --outDir ./static
```

Comment:
These should probably be condensed into one yarn command, in the `package.json` `scripts`

Reply:
The existing `build` command from the scripts already does the job. Also changed to using the default `dist` output directory for the static build